includes/WikiManager.php

```diff
@@ -5,17 +5,16 @@
 
 class WikiManager {
 	private $dbname = null;
-	private $dbw = null;
```

Comment:
Why?

Reply:
It's not used from what i can tell.

Reply:
Yes it is?

Reply:
Where? Searching the file i see no more occurrences of $dbw?

Reply:
Line 55

Reply:
Oh damn. Ok putting it back.